src/main/java/com/google/sps/data/Election.java

```diff
@@ -104,31 +104,35 @@ public static Election fromElectionQuery(JSONObject electionQueryData) throws JS
   public Election fromVoterInfoQuery(DatastoreService datastore, JSONObject voterInfoQueryData)
       throws JSONException {
     Set<Long> contestKeyList = this.getContests();
-    Set<Long> propositionKeyList = new HashSet<>();
+    Set<Long> referendumKeyList = new HashSet<>();
     if (voterInfoQueryData.has("contests")) {
       JSONArray contestListData = voterInfoQueryData.getJSONArray("contests");
       for (Object contestObject : contestListData) {
         JSONObject contest = (JSONObject) contestObject;
 
-        long contestEntityKeyId =
-            Contest.fromJSONObject(datastore, contest).addToDatastore(datastore);
-        contestKeyList.add(contestEntityKeyId);
+        // Referendums are a separate contest type, so separate them out from the office positions.
+        if (contest.getString("type").equals("Referendum")) {
```

Comment:
Maybe put "referendum" into an enum instead of hard coding? Then we can also use a switch here.

Reply:
@rogerlaucorp can enums be given String values? Or would it be better to use a static constant here?

Reply:
You could do something like this:
https://howtodoinjava.com/java/enum/java-enum-string-example/
or like this:
https://www.programiz.com/java-programming/enum-string